### ISSUE TITLE
error message  in instant repl,  importing namespace

### ISSUE THREAD

**Issue by mikelemus27** [oakes/Nightcode#255]
im new in clojure, so maybe this  is  not important.
first an image

![nightcodeerror](https://user-images.githubusercontent.com/146947/54293511-badc4a00-4575-11e9-8f81-b4fe2e6bed6a.JPG)

in niightcode 2.60  i get this  error  in  instant repl

but   it doesnt afect the  execution

 i opened it in  nighlight editor, and  on it, is  working  good.
![nightlight ok](https://user-images.githubusercontent.com/146947/54293999-8026e180-4576-11e9-9117-4fc7f529caf4.JPG)

thats  all

**Comment by oakes** [comment]
Yep you're not doing anything wrong, it's just a limitation of Nightcode. The instarepl basically only works with clojure core because it is running separately from your project. Nightlight doesn't have this limitation because it is running inside your project so it has access to everything.
